pkgs/applications/misc/blender/derivation.nix

```diff
@@ -0,0 +1,280 @@
+{ config, stdenv, lib, fetchFromGitea, fetchzip, boost, cmake, ffmpeg, gettext, glew
+, ilmbase, libepoxy, libXi, libX11, libXext, libXrender
+, libjpeg, libpng, libsamplerate, libsndfile
+, libtiff, libwebp, libGLU, libGL, openal, opencolorio, openexr, openimagedenoise, openimageio, openjpeg
+, openvdb, libXxf86vm, tbb, alembic
+, zlib, zstd, fftw, opensubdiv, freetype, jemalloc, ocl-icd, addOpenGLRunpath
+, jackaudioSupport, libjack2
+, cudaSupport, cudaPackages ? { }
```

Comment:
`cudaPackages` is always defined, no need for `? { }`

Reply:
This is part of the original file. Will remove.